rice-framework/krad-data/src/test/java/org/kuali/rice/krad/data/jpa/JpaMetadataProviderTest.java

```diff
@@ -153,12 +143,12 @@ public void testGetMetadataForCollection() {
 		assertNotNull("Collections object should not be null", collections);
 		assertEquals("Collections size incorrect", 4, collections.size());
 
-		DataObjectCollection collection = collections.get(0);
+		DataObjectCollection collection = metadata.getCollection("collectionProperty");
 		assertEquals("property name incorrect", "collectionProperty", collection.getName());
```

Comment:
This relates to the bullet in the PR's description where there were some ordering issues in the tests where things come back in a different order.  I'm also not inclined to clean up these tests more than necessary because if I started fixing issues like making "collectionProperty" a variable I'd never be able to stop ;).